sequenceSearch/sequencetools.py

```diff
@@ -40,7 +40,15 @@ def blast(query, db="/clusterCAD/pipeline/data/blast/clustercad_subunits", evalu
                                          outfmt=5,
                                          num_threads=2,
                                          max_target_seqs=max_target_seqs)
-    result, stderr = blastp_cline(stdin=queryFasta)
+    
+    try: #Changed by Yifei
```

Comment:
Let's leave the exception here, instead of returning 0, and then catch the exception one level up in views.